src/components/study-pane.js

```diff
@@ -175,6 +180,22 @@ const StudyPane = () => {
         history.replace("/studies/" + studyName)
     }
 
+    const handleUpdateSwitchState = useCallback( (studyName, breakerId, open, svgUrl, svgDisplayInfo) => {
+        updateSwitchState(studyName, breakerId, open).then( response => {
+            if (response.ok) {
+                setCurrentSvg(fetchSvg(svgUrl));
+                setSvgDisplayInfo(svgDisplayInfo);
```

Comment:
svgDisplayInfo should be local to the function that forces the refresh of an svg in the sld